src/components/intellectual_property_list/IntellectualPropertySummary.tsx

```diff
@@ -58,39 +60,48 @@ export const IntellectualPropertySummary: React.FC<IntellectualPropertySummaryPr
         )
       })}
       <div className="collaborator-summary-edit-delete-buttons">
-        <button
-          type="button"
-          style={{ marginLeft: 10, marginRight: 10, ...buttonStyle }}
+        {/* view button */}
+        <a
```

Comment:
Should this really be a button?

Reply:
I decided to use anchors because they look better, and I can assign them the role `button` for screen readers.

**Button**
<img width="65" height="50" alt="Button" src="https://github.com/user-attachments/assets/3726e941-2425-4eb9-ac88-f429e4d34381" />

**Anchor**
<img width="65" height="50" alt="Anchor" src="https://github.com/user-attachments/assets/9d42bcf5-4f60-4276-8c09-8e23c2340cdb" />